ShoppingAssistantWebClient.Web/Pages/Cart.cshtml.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace ShoppingAssistantWebClient.Web.Pages
+{
+    public class CartModel : PageModel
+    {
+        public List<Product> products = new List<Product> {
+                new Product {Description = "HDMI cabel HDMI cabel HDMI cabel HDMI cabel HDMI cabel HDMI cabelHDMI cabel", Rating = 4.0, Price = 12},
+                new Product {Description = "super mega hdmi cabel", Rating = 3.8, Price = 13.11},
+                new Product {Description = "", Rating = 4.0}
+        };
+    
+        public void OnGet()
+        {
+
+        }
+    }
+
+    public class Product {

```

Comment:
Move this to the separate file in the models folder. 1 class = 1 file - it's the rule. And name it ProductModel. Also add spacing between each property